huxley/www/js/components/DelegationAttendanceRow.js

```diff
@@ -11,8 +11,17 @@ var DelegationAttendanceRow = React.createClass({
   propTypes: {
     onChange: React.PropTypes.func,
     countryName: React.PropTypes.string,
-    countryID: React.PropTypes.string,
-    delegates: React.PropTypes.array,
+    assignmentID: React.PropTypes.number,
+    attendance: React.PropTypes.object,
+  },
+
+  shouldComponentUpdate(nextProps, nextState) {
```

Comment:
I added a check on assignmentID; that was an oversight not to include form the start. We should have some sort of check on should it update though since for a large committee there may be hundreds of these rows and I think it would be inefficient to re-render every time one is updated.